apps/client/src/app/pages/portfolio/x-ray/x-ray.module.ts

```diff
@@ -0,0 +1,26 @@
+import { GfRulesModule } from '@ghostfolio/client/components/rules/rules.module';
+import { GfFireCalculatorComponent } from '@ghostfolio/ui/fire-calculator';
+import { GfPremiumIndicatorComponent } from '@ghostfolio/ui/premium-indicator';
+import { GfValueComponent } from '@ghostfolio/ui/value';
+
+import { CommonModule } from '@angular/common';
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { NgxSkeletonLoaderModule } from 'ngx-skeleton-loader';
+
+import { XRayComponent } from './x-ray.component';
+
+@NgModule({
+  declarations: [XRayComponent],
+  imports: [
+    CommonModule,
+    GfFireCalculatorComponent,
+    GfPremiumIndicatorComponent,
+    GfRulesModule,
+    GfValueComponent,
+    NgxSkeletonLoaderModule,
+    RouterModule.forChild([{ path: '', component: XRayComponent }])
```

Comment:
Could you create `x-ray-page-routing.module.ts` (see `fire-page-routing.module.ts`) instead of this?